### ISSUE TITLE
If run deauth and beacon together, then nothing will work

### ISSUE THREAD

**Issue by ivsbex** [spacehuhn/esp8266_deauther#1440]
**Describe the bug**
If run deauth and beacon, both of attacks will be stopped and you can't even run single deauth or single beacon (probe too)

**To Reproduce**
1. Go to 'Attacks'
2. Click on 'Start' when 'Deauth'
3. Click on 'Start' when 'Beacon'

**Expected behavior**
Deauth and beacon will run together, or you can run one of them again

**Screenshots**
Can't show on screenshots, only video
Link on YouTube: https://www.youtube.com/watch?v=MBHVSxwotUE

**Environment (please complete the following information):**
 - OS: Windows
 - Version: 10 1909 (18363.1556) x64
 - Browser: Chrome (Newest version)
 - Hardware: ESP8266 CH340 NodeMCU v3 with USB
![image](https://user-images.githubusercontent.com/60809389/120932984-be34f080-c700-11eb-92ee-38e00ca42281.png)

**Additional context**
Sorry for my bad English

**Comment by ghost** [comment]
yes i have the same problem with you, when running both simultaneously (Deauth +Beacon) the program stops itself....

Any help for the problem this ?

**Comment by 4Powerr** [comment]
I have this taste of the problem. I tried to upload older software, maybe it worked correctly on older ones.
